GAE/src/org/waterforpeople/mapping/analytics/dao/SurveyQuestionSummaryDao.java

```diff
@@ -124,6 +117,11 @@ public static synchronized void incrementCount(QuestionAnswerStore answer,
      */
     @SuppressWarnings("unchecked")
     public List<SurveyQuestionSummary> listByResponse(String questionId, String questionResponse) {
+        //Reject too-long key; TODO truncate instead?
```

Comment:
I suggest lets go with the truncate option on this one.

Reply:
Ok.

Reply:
Can as well clear out the comment

Reply:
done